aws/sdk/gradle.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+# base set of services that are generated unless other options are specified:
+aws.services.tier1=+apigateway,+applicationautoscaling,+autoscaling,+autoscalingplans,+batch,+cloudformation,+cloudwatch,+cloudwatch,+cloudwatchlogs,+cognitoidentity,+cognitoidentityprovider,+cognitosync,+config,+dynamodb,+ebs,+ec2,+ecr,+ecs,+eks,+iam,+kinesis,+kms,+lambda,+medialive,+mediapackage,+polly,+qldb,+qldbsession,+rds,+rdsdata,+route53,+s3,+sagemaker,+sagemakera2iruntime,+sagemakeredge,+sagemakerfeaturestoreruntime,+secretsmanager,+sesv2,+snowball,+sns,+sqs,+ssm,+sts
```

Comment:
Properties files allow you to break things over multiple lines if you want to make this more readable:
```
aws.services.tier1=\
  +apigateway,\
  +applicationautoscaling,\
...
```